locksmith/src/controllers/metadataController.ts

```diff
@@ -21,7 +21,7 @@ namespace MetadataController {
       ) {
         defaultResponse.name = lockMetadata.name
         defaultResponse.description = lockMetadata.description
-        defaultResponse.image = lockMetadata.image
+        defaultResponse.image = lockMetadata.image || defaultResponse.image
```

Comment:
Sometimes we do not have a logo to use... so we default to the Unlock default logo.